### DISCUSSION TITLE
Instagram

### DISCUSSION THREAD

**Discussion by AllanNobi** [AmauriC/tarteaucitron.js#1281] [Q&A]
Hi,

In order to comply with W3C standards, could you add “data-” in front of the attribute names in **tarteaucitron.services.js** file?
For example, instead of having <div class=“instagram_post” postID=“postID” width=“width” height=“height”></div>
Ask: <div class=“instagram_post” data-postid=“postID” data-width=“width” data-height=“height”></div>

This applies to other memory services.

In the transmission of the postID I had to add the value of the previous level, which can be “p”, “reel” or “tv”.
Example: “reel/postID”.

Have a nice day!

**Comment by AmauriC** [comment]
Hi

You can already add `data-` before attributes, it will work.
